examples/with-typescript/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": false,
+        "sourceMap": false,
```

Comment:
might be best to make this file as simple as possible, the bare minimum to compile successfully. let other folks choose their compiler flags as necessary. 

just set `jsx`, `module`, and `target`.